plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -722,11 +744,10 @@ enterComputeCek = computeCek (toWordArray 0) where
             RuntimeSchemeArrow schB -> case f arg of
                 Left err -> throwKnownTypeErrorWithCause argTerm err
                 Right y  -> do
-                    -- TODO: should we bother computing that 'ExMemory' eagerly? We may not need it.
+                    -- TODO: should we compute @toExMemory arg@ eagerly instead? We may not need it.
                     -- We pattern match on @arg@ twice: in 'readKnown' and in 'toExMemory'.
                     -- Maybe we could fuse the two?
-                    let runtime' = BuiltinRuntime schB y . exF $ toExMemory arg
-                    res <- evalBuiltinApp fun term' runtime'
+                    res <- evalBuiltinApp fun term' schB y $! exF (toExMemory arg)
```

Comment:
Same strictness as before, since before `evalBuiltinApp` would force `runtime'` having a strict `exF (toExMemory arg)` inside of it.